app/styles/_notifications.less

```diff
@@ -31,6 +42,9 @@ notification-drawer-wrapper {
       }
     }
   }
+  .drawer-pf-close, .drawer-pf-toggle-expand {
+    padding: 2px 10px;
+  }
```

Comment:
We should give feedback to patternfly on things like this. Shouldn't have to be overriding  these settings.

Reply:
@sg00dwin why are we doing this?  Agree that we shouldn't be overriding things.  If we need PF to change something let me know.  If they cannot do it in time, then we can override with a plan to change later.

Reply:
@jeff-phillips-18 @serenamarie125 Makes sense. For context, @sg00dwin added a few pixels of extra padding between the icon and the edge of the drawer.

@sg00dwin It might just be easier to remove this.

Before:

![openshift web console 2017-09-01 14-40-35](https://user-images.githubusercontent.com/1167259/29983390-8f2bc71e-8f23-11e7-8fbd-fe733e62b2e0.png)

After:

![openshift web console 2017-09-01 14-45-18](https://user-images.githubusercontent.com/1167259/29983499-2fa3ec3a-8f24-11e7-82c2-6c03b8616fd9.png)

Reply:
Felt the click target as too small,  5px padding between icon and edge.
I'll revert and suggest issue upstream.

before
<img width="32" alt="screen shot 2017-09-01 at 2 45 35 pm" src="https://user-images.githubusercontent.com/1874151/29983732-34e8c746-8f25-11e7-9d3f-e751304e541e.png">

after
<img width="40" alt="screen shot 2017-09-01 at 2 46 28 pm" src="https://user-images.githubusercontent.com/1874151/29983748-3d4f47ca-8f25-11e7-9538-71250234c024.png">

Reply:
> We should give feedback to patternfly on things like this. Shouldn't have to be overriding these settings.

While I don't disagree, sometimes it's infinitely easier just to fix a problem in the code that you control.

Reply:
Totally agree that what you have done looks much better!  

I do think that if we take those liberties, please call them out so that UX can review and we can make a plan get them into PatternFly for consistency :D  Understand your point @rhamilto but when our customers are dealing with a portfolio of products, everyone doing their own things just causes further issues.

Reply:
+1 prob agree to both...temp fix here then ensure an issue upstream to track/migrate it.